### DISCUSSION TITLE
NextCloudPi LXC, unable to access shell

### DISCUSSION THREAD

**Discussion by c0bnut** [tteck/Proxmox#925] [Help Needed] [answered]
**Describe the issue**
When trying to access the LXC via shell or SSH you get the error message "This account is currently not available" and are immediately logged out. I have tried both the root account and ncp account and they both return the same error message. This appears to be a known issue with NextCloudPi [see this forum post](https://help.nextcloud.com/t/nextcloudpi-account-currently-not-available/148674) however I can't do the fix as SSH appears to be missing from the NCP admin panel.

**Screenshots**
![image](https://user-images.githubusercontent.com/36831111/212047418-523ae663-77f6-489d-aeb6-e4f9ebd5b700.png)

**Comment by rollingafull** [comment]
Confirm this issue.

**Comment by tteck** [comment]
https://github.com/nextcloud/nextcloudpi/issues/1693

**Comment by tteck** [comment, marked as answer]
### Script has been fixed to allow shell access.

#### To fix your existing LXC, in the Proxmox Shell run the commands below

`pct exec CTID bash` (change **CTID** to your Container ID)
`sed -i '{s|root:/usr/sbin/nologin|root:/bin/bash|g}' /etc/passwd`
`reboot`

**Comment by jokerigno** [comment]
hi @tteck maybe it's just me but even using the fix I'm unable to see ssh in ncp panel. I used pct exec CTID bash command to change root password and then use winscp (I wrote just for someone else that may found this useful).

Maybe I'm missing some step?
